docker-compose.yml

```diff
@@ -59,7 +59,7 @@ services:
     image: backend:latest
     command: bundle exec sidekiq -C config/sidekiq.yml
     volumes:
-      - ./backend:/backend:cached
```

Comment:
Why do we need this change for the language switcher?

Reply:
> Why do we need this change for the language switcher?

Hm, i guess nope. I thought that only local changes... I repair that